### ISSUE TITLE
holt_laury 2

### ISSUE THREAD

**Issue by bissettp** [expfactory/expfactory-experiments#193]
![screen shot 2016-02-08 at 2 16 17 pm](https://cloud.githubusercontent.com/assets/9681335/12901153/e46c4752-ce6e-11e5-87b3-1597eac252f5.png)

Unless I'm misunderstanding the task, something seems to be wrong with the comparisons. The right choices are objectively better than the left choices in every way in all 10 cases. As currently constructed, this task does not seem to test risk aversion as much as whether subjects understand the most basic principles of gambling (i.e., pick a more advantageous choice or a less advantageous choice).

**Comment by zenkavi** [comment]
Here is the original task: http://users.nber.org/~rosenbla/econ311-04/syllabus/holtlaury.pdf

I am not sure what you mean by "objectively better." The expected value of the right gambles exceeds that of the left gambles after the fifth choice. The risk attitude then is indicated by how long people persist to choose the left gambles after this point.

The one I have coded up is an 50X hypothetical version with percentages that I have used previously in online experiments and have seen used by others. Whether it is measuring the construct of "risk aversion" is a theoretical debate, on which multiple people (including me) are trying to get papers out. Currently though with its ~3500 citations this is considered a standard measure of risk attitudes.
